### PR TITLE
Made the page load dynamic

### PR DESCRIPTION
**Pull request by techryon**
Html file has only the skeleton info
CSS now controls the size of the cards being displayed
JS loads the default image

Ignore the files which says newApproach - Its my sandbox files to fiddle things

### CODE REVIEW

File: guesscard-index.html [hunk 1]
```diff
@@ -12,185 +12,189 @@
...
+    <!--Div to hold each image.
+    Used two divs might be needed later. So leaving it for now-->
       <div class="cell cell-1">
-        <div class="check">
-          <img src="./res/images/card-background.png" class="cell__img">
+        <div>
+          <img class="cell__img" id=cell__img1 src="">
         </div>
       </div>
       <div class="cell cell-2">
```
Comment (md-weber) [inline guesscard-index.html]: Mhhh this whole cell thing looks somehow like repetition, how could we do that in a Javascript way?

Comment (md-weber) [reply guesscard-index.html]: Maybe a loop could help us here?

File: res/scripts/newApproach.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+//  for (var faceCard=0; faceCard<=51; faceCard++){
+ //   faceCardsArray[faceCard] = new Image(); //Every element of the array is defined as a new image
+   // faceCardsArray[faceCard].src = "./res/images/face/" + faceCard + ".png"; //every element source has been set
+
+  //}
+ 
+});
+
+
+function onload_image() {
```
Comment (md-weber) [inline res/scripts/newApproach.js]: Puh hard to read PR :) so the new Approach is the correct one? I would consider remove the old approach and never call something new. 
Additionally you do not have to save "old" stuff, as soon as you have it in Git there exists always a revision of it.

File: res/scripts/script-index.js [hunk 3]
```diff
@@ -1 +1,18 @@
...
+//AddEventListener to DOMContentLoaded, so the image waits to load
+//after the page is loaded
+document.addEventListener("DOMContentLoaded", function(event) {
+	onload_image();
+ 
+});
+
+//Function to load default images. Looping it so it loads the same image in 36cells
+//This function gets the image using Id and adds the source attributes 
+function onload_image() {
```
Comment (md-weber) [inline res/scripts/script-index.js]: Where does this now belong to?

Comment (md-weber) [reply res/scripts/script-index.js]: Looks very much the same as the file above, maybe merge them?

File: res/styles/style-index.css [hunk 4]
```diff
@@ -4,16 +4,16 @@ h1 {
 
 .container {
     display: grid;
-    grid-template-columns: repeat(9, 9.5vw);
-    grid-template-rows: repeat(3, 12.5vw);
-    grid-gap: 3px;
+    grid-template-columns: repeat(9, 100px);
+    /*grid-template-rows: repeat(3, 9.5vw12.5vw);
```
Comment (md-weber) [inline res/styles/style-index.css]: Remove comments you have them anyway in the revision of git, if you need them in the future. Only leave comments with explanation of the current code inside.

**Review by md-weber — CHANGES_REQUESTED** [review]

File: res/scripts/newApproach.js (hunk 2 above)
Comment (techryon) [reply res/scripts/newApproach.js]: Sorry this was a sandboxy file... renamed it to donot-review

File: guesscard-index.html (hunk 1 above)
Comment (techryon) [reply guesscard-index.html]: implemented looping from javascript, now the divs and img are created from script

File: res/styles/style-index.css (hunk 4 above)
Comment (techryon) [reply res/styles/style-index.css]: Commented codes removed

File: guesscard-index.html [hunk 5]
```diff
@@ -12,190 +12,11 @@
...
-      <div class="cell cell-36">
-        <div>
-          <img src="./res/images/card-background.png" class="cell__img">
-        </div>
-      </div>
-              
-    </div>
-
+    <!--Div container to hold the grid of 36 cards-->
+    <div class="container" id="container">
```
Comment (md-weber) [inline guesscard-index.html:16]: Your browser will understand but please always add an ending tab if necessary.
```suggestion
    <div class="container" id="container"></div>
```

File: res/scripts/script-index.js [hunk 6]
```diff
@@ -1 +1,28 @@
...
+document.addEventListener("DOMContentLoaded", function(event) {
+ // onload_image();
+  createDiv();
+ 
+});
+
+  //Function to create the div elements for the grid
+  function createDiv() {
+    var parentDiv = document.getElementById("container");
+    for(i=1; i<=36; i++){
```
Comment (md-weber) [inline res/scripts/script-index.js]: Careful with global variables.
```suggestion
    for(let i=1; i<=36; i++){
```
http://tobyho.com/2011/10/25/js-accidental-global-variables/

File: res/styles/style-index.css [hunk 7]
```diff
@@ -1,33 +1,18 @@
+/*Setting the color for h1*/
 h1 {
     color: SeaGreen;
   }
 
+/*Setting the grid for the container class*/
```
Comment (md-weber) [inline res/styles/style-index.css:6]: Usally I like comments, but the code here is very obvious, so do you think you need the comment? Is it add value?
```suggestion

```

**Review by md-weber — APPROVED** [review]
Cool much better!

File: res/scripts/script-index.js (hunk 6 above)
Comment (md-weber) [reply res/scripts/script-index.js]: Maybe consider renaming also i to something better, like index.

File: guesscard-index.html (hunk 5 above)
Comment (techryon) [reply guesscard-index.html:16]: OMG, i accidentally deleted while removing the 36 divs :)

File: res/scripts/script-index.js (hunk 6 above)
Comment (techryon) [reply res/scripts/script-index.js]: Fixed :)

File: res/styles/style-index.css (hunk 7 above)
Comment (techryon) [reply res/styles/style-index.css:6]: hmm, i'll leave it for a while, until i get used to it...
Sometimes i get confused with what i did on the code...
Also only now i realize the importance of comments

**Comment by techryon** [comment]
Implemented the comments
Merging now

**Pull request merged**